src/rust/src/utils/mod.rs

```diff
@@ -748,18 +748,15 @@ pub fn robj_to_datatype(robj: extendr_api::Robj) -> RResult<RPolarsDataType> {
     Ok(RPolarsDataType(ext_dt.0.clone()))
 }
 
+// TODO: move to R side?
```

Comment:
Can this be done in this PR? I think it's just a matter of removing the Rust part and call `parse_as_polars_duration_string()` in each R function with a duration input, right?

Reply:
If we remove it, we would also need to implement on the R side the conversion to `Duration` (not `Duration` string) (which is called in the following location), which would require quite a bit of additional work.
I do not have time to do that right now.
https://github.com/pola-rs/r-polars/blob/a028047550c8bf5ad23a75d9c02f8c3825b7beb2/src/rust/src/rlib.rs#L102

Reply:
Perhaps it would be more appropriate to convert the difftime object directly to a Duration type rather than converting it once to a String.

Reply:
>  I do not have time to do that right now.

Alright, let's not include it in this PR, I'll see if I have some time later for that